requirements.txt

```diff
@@ -1,2 +1,3 @@
-websockets
-aiohttp
\ No newline at end of file
+aiohttp
```

Comment:
Can you put specific more versions?

Reply:
Yeah, I added the version number for all of the libraries we use.